virttest/env_process.py

```diff
@@ -1255,6 +1255,25 @@ def preprocess(test, params, env):
         version_info["vm_bootloader_version"] = str(vm_bootloader_ver)
         LOG.debug("vm bootloader version: %s", vm_bootloader_ver)
 
+    # Checking required virtio-win version, if not satisfied, cancel test
+    if params.get("required_virtio_win") or params.get("required_virtio_win_prewhql"):
+        if params.get("cdrom_virtio"):
+            cdrom_virtio = params["cdrom_virtio"]
+            virtio_win_range = params.get("required_virtio_win_prewhql") if re.search(
+                'prewhql', cdrom_virtio) else params.get("required_virtio_win")
```

Comment:
Suggest checking this after get actual iso name.

Reply:
Okay, it will be better.Thanks!